configs/sst_networking-core-network-management.yaml

```diff
@@ -0,0 +1,52 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Core Network Management
+  description: The Core Network Management tools for the local host. This includes NetworkManager and related tools.
+  maintainer: sst_networking
+  packages:
+  - ModemManager
```

Comment:
Thanks. Sorting the list alphabetically might make it easier to lookup packages.

Reply:
They are sorted, with `lang C` and vim's sort. Granted, that is case-sensitive. I wasn't aware that another locale would be preferable.

Reply:
the locale is good, case-insensitive would be nice.